tests/PerformanceHarness/performance_test_basic.py

```diff
@@ -124,6 +124,7 @@ def __post_init__(self):
 
             def configureValidationNodes():
                 validationNodeSpecificNodeosStr = ""
+                validationNodeSpecificNodeosStr += '--p2p-accept-transactions false '
```

Comment:
Why not remove the statement above and do `validationNodeSpecificNodeosStr = '--p2p-accept-transactions false ' directly?

Same comment for below.

Reply:
You don't have to make the changes. I am approving it now so that it can make to 5.0.

Reply:
That would be a fine change.  Previously all additions to `validationNodeSpecificNodeosStr` were conditional so it was needed to initialize the variable.  Looks like down below `apiNodeSpecificNodeosStr` could've been initialized directly with the first option instead of to a blank string first.